code/rewriter.go

```diff
@@ -159,9 +160,7 @@ func (r *Rewriter) rewriteExpr(expr ast.Expr) error {
 		return r.rewriteFuncLit(ex)
 
 	case *ast.CompositeLit:
-		if err := r.rewriteExpr(ex.Type); err != nil {
```

Comment:
This case is needless, I remove them because the `Type` cannot be the `Expression`, We don't need to check and rewrite it.

Reply:
OK